### PR TITLE
#166413869 Convert project from es5 to es6

### PR DESCRIPTION
**Pull request by deschantkn**
#### What does this PR do?
Configures project to use `es6` 

#### Description of Task to be completed?
* Convert project from `es5` to `es6`
* Reconfigure project structure
* Switch package manager from `npm` to `yarn`

#### How should this be manually tested?
Checkout this branch and attempt to start the server

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#166413869

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A

### CODE REVIEW

**Review by rukundoeric — APPROVED** [review]

**Review by eliemugenzi — APPROVED** [review]

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env"]
+}
```
Comment (Diama1) [inline .babelrc]: here you should put a new line after the last line of code.

**Review by Diama1 — APPROVED** [review]

**Review by EmyRukundo — APPROVED** [review]
good work

**Review by Quantum-35 — CHANGES_REQUESTED** [review]
Any Reasons why You deleted the two files in `config` Folder?

**Comment by deschantkn** [comment]
> Any Reasons why You deleted the two files in `config` Folder?

They were written `es5`, I thought we could re-write in `es6` when we would need to use them

**Review by Quantum-35 — APPROVED** [review]

**Pull request merged**